community/kernel/src/main/java/org/neo4j/kernel/api/DataRead.java

```diff
@@ -80,7 +79,7 @@ PrimitiveLongIterator nodesGetFromIndexScan( IndexDescriptor index )
      * Returns node id of unique node found in the given unique index for value or
      * {@link StatementConstants#NO_SUCH_NODE} if the index does not contain a
      * matching node.
-     *
+     * <p/>
```

Comment:
When setting the language level to 1.8, Intellij starts inserting ```<p>``` instead of ```<p/>``` (also converts on auto-format). Might be an option in the future. 
@cleishm well, it is helpful, otherwise the rendered output doesn't have the blank line :-)